wiki/Client/Beatmap_editor/Timing/en.md

```diff
@@ -1,25 +1,96 @@
+**Timing** is the [beatmap editor](/wiki/Client/Beatmap_editor)'s tab for changing and setting up timing of a [beatmap](/wiki/Beatmap), which is essential for correct song representation. It contains related settings and helpers, and also a [dedicated window](#timing-setup-panel) for working with multiple [timing points](#timing-points), both for musical structure description and beatmap design purposes.
 
-## Timing and Control Points window
+## Timing points
 
-![Screenshot of the Timing and Control Points window](/wiki/shared/timing/TimingSetup.png)
+*See also: [Offset](/wiki/Offset)*
 
-Timing points can be added from the `Timing and Control Points` window in the editor (default shortcut `F6`). Timing points divide a song into timing sections to accurately reflect changes to a song's timing.
+In [mapping](/wiki/Beatmapping), a *timing point*, colloquially called an *offset*, is a way to apply common settings, such as [timing](/wiki/Beatmapping/Timing), [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) multiplier, or [hitsounds](/wiki/Beatmapping/Hitsound)' volume, to a whole section of the beatmap. In osu!, there are two types of timing points.
 
 ### Uninherited timing point
 
-An uninherited timing point, displayed as a red line, is the base for a timing section's BPM, offset, and meter. Every beatmap must have at least one red line, however some songs require multiple due to more complex composition or timing inaccuracies caused by live performance.
+::: Infobox
+![](img/uninherited-offsets.png "Several uninherited timing points located in the timing setup panel")
+:::
+
+An **uninherited** timing point has its own timing settings. Several such points are used to describe timing changes in the song, such as [tempo](/wiki/Music_theory/Tempo) or beat irregularities, or different size of the [time signature](/wiki/Music_theory/Time_signature). Adding a new uninherited timing point resets the metronome, and makes it count beats from the timing point's offset, specified in milliseconds.
+
+Uninherited timing points have red colour on the bottom timeline and in the [timing setup panel](#timing-setup-panel). Due to this, they are often called "red offsets", or "red lines".
 
 ### Inherited timing point
 
-An inherited timing point, displayed as a green line, inherits BPM, offset, and meter settings from the previous red line. Green lines are used to adjust [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) and [hitsound](/wiki/Beatmapping/Hitsound) settings.
+::: Infobox
+![](img/inherited-offsets.png "Several inherited timing points with different slider velocity multiplier, volume, and kiai settings")
+:::
+
+An **inherited** offset uses currently active timing settings, *inheriting* them from the previous timing point. Inherited timing points are used for the following purposes:
+
+- Changing [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) on a certain section of the beatmap
```

Comment:
```suggestion
- Changing [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity)
```
none of the other use cases mention a specific section even though they all can do that, might be best to remove?